core/src/main/resources/jenkins/security/ApiTokenProperty/config.jelly

```diff
@@ -103,12 +103,12 @@ THE SOFTWARE.
                                         </span>
                                     </j:otherwise>
                                 </j:choose>
-                                    
+
                                 <a href="#" onclick="return revokeToken(this)" class="token-revoke"
                                    data-message-if-legacy-revoked="${%RevokedToken}"
                                    data-confirm="${%ConfirmRevokeSingle}"
                                    data-target-url="${descriptor.descriptorFullUrl}/revoke">
-                                    <l:icon class="icon-text-error icon-sm" alt="${%RevokeToken}" />
+                                    <st:include page="/lib/layout/svg-icon-token-revoke.jelly" />
```

Comment:
Did this deliberately drop the alt text?

Reply:
Yes, because inline svg's don't use alt attributes. But you are right, we should keep the functionality so I added a title to the revoke svg and a title to the copy button so that a text shows up on mouse hover.